.github/workflows/black-action.yml

```diff
@@ -0,0 +1,28 @@
+name: black-action
+
+on: [push, pull_request]
+
+jobs:
+  linter_name:
+    name: runner / black
+    runs-on: ubuntu-latest
+    if: github.head_ref==github.base_ref # Don't run when target is remote
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check files using the black formatter
+        uses: rickstaa/action-black@v1
+        id: action_black
+        with:
+          black_args: "."
+      - name: Create Pull Request
+        if: steps.action_black.outputs.is_formatted == 'true'
+        uses: peter-evans/create-pull-request@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          title: "Format Python code with psf/black push"
+          commit-message: ":art: Format Python code with psf/black"
+          body: |
+            There appear to be some python formatting errors in ${{ github.sha }}. This pull request
+            uses the [psf/black](https://github.com/psf/black) formatter to fix these issues.
```

Comment:
I suggest a slightly milder wording here, like "It appears that some Python code in ${{ github.sha }} is not formatted according to the [Black](https://github.com/psf/black) specification. This pull request formats the code in question."